lib/beaker/host/unix/exec.rb

```diff
@@ -302,7 +302,7 @@ def ssh_service_restart
   #   (from {#ssh_service_restart}).
   def ssh_permit_user_environment
     case self['platform']
-    when /debian|ubuntu|cumulus|huaweios|archlinux|el-|centos|fedora|redhat|oracle|scientific|eos|opensuse|sles|solaris/
+    when /amazon|debian|ubuntu|cumulus|huaweios|archlinux|el-|centos|fedora|redhat|oracle|scientific|eos|opensuse|sles|solaris/
```

Comment:
That does make sense, but not sure how many changes to defaults I wanted to do outside of the addition itself. So your saying for the functions that involved a change, they should default to the correct systemd/systemctl command instead of raising an error?

Reply:
Opened https://github.com/voxpupuli/beaker/issues/1825 to address that.